library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -1275,6 +1301,22 @@ public boolean changeLocationUpdates(long updateInterval, long fastestUpdateInte
         return false;
     }
 
+    @Override
+    public boolean isKujakuLayerAdded(@NonNull KujakuLayer kujakuLayer) {
+        String[] layerIds = kujakuLayer.getLayerIds();
+        if (mapboxMap != null && mapboxMap.getStyle() != null && mapboxMap.getStyle().isFullyLoaded()) {
+            for (String layerId: layerIds) {
```

Comment:
Validate layerIds is not null?

Reply:
LayerIds will never be null. The correct annotation to enforce that is in the interface for the `KujakuLayer` and all implementations provide non-null results

Reply:
OK for this case.

But keep in mind that annotations are merely informational, they don't necessarily provide validation.

No change required here.